k8s/namespaces/ia/ia-case-payments-api/demo.yaml

```diff
@@ -2,12 +2,15 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ia-case-payments-api
+  namespace: ia
   annotations:
```

Comment:
are `annotations `needed?

Reply:
I don't think its relevant, have removed it.